include/kodlab_mjbots_sdk/mjbots_control_loop.h

```diff
@@ -125,10 +126,12 @@ class MjbotsControlLoop : public AbstractRealtimeObject {
   void SafeProcessInput();
 
   /*!
-   * @brief virtual class to be implemented when logging. Process data in m_lcm_sub.m_data; This function is threadsafe
-   * and won't run if the LCM thread holds the mutex
+   * @brief virtual class to be implemented when logging. Process data in
```

Comment:
function not class

Reply:
function member or method I guess

Reply:
Amended in e73960cbc50c2c7567521410a70b84a7d077c952